podcasts/MiniPlayerViewController+Positioning.swift

```diff
@@ -54,8 +54,6 @@ extension MiniPlayerViewController {
 
             playerOpenState = .animating
 
-            fullScreenPlayer.modalPresentationStyle = .overCurrentContext
```

Comment:
This is not part of the fix, but it was here due to a bad merge (although it apparently didn't caused any issues)